erpnext/projects/doctype/task/task.py

```diff
@@ -202,7 +202,7 @@ def set_tasks_as_overdue():
 	tasks = frappe.get_all("Task", filters={'status':['not in',['Cancelled', 'Closed']]})
 	for task in tasks:
 		if frappe.db.get_value("Task", task.name, "status") in 'Pending Review':
-			if getdate(frappe.db.get_value("Task", task.name, "review_date")) < getdate(today()):
+			if getdate(frappe.db.get_value("Task", task.name, "review_date")) > getdate(today()):
```

Comment:
If you get all the fields in `get_all`, it'll save a lot of `db.get_values` here, so its a no brainer change to make.

Reply:
Done